pkg/network/monitoring.go

```diff
@@ -0,0 +1,31 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/monitoring"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+)
+
+func renderMonitoring(manifestDir string, clusterInfo *ClusterInfo) ([]*unstructured.Unstructured, error) {
+	if !clusterInfo.MonitoringAvailable {
+		return nil, nil
+	}
+
+	// render the manifests on disk
+	data := render.MakeRenderData()
+	data.Data["Namespace"] = os.Getenv("OPERAND_NAMESPACE")
+	data.Data["MonitoringNamespace"] = monitoring.GetMonitoringNamespace()
+	data.Data["MonitoringServiceAccount"] = monitoring.GetMonitoringServiceAccount()
+
+	objs, err := render.RenderDir(filepath.Join(manifestDir, "monitoring"), &data)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to render monitoring manifests")
+	}
+
+	return objs, nil
+}
```

Comment:
This has nothing to do with "network" package, it belongs to "monitoring" package so we do `monitoring.Render`

Reply:
well the entire Render function is in `network.go`
I moved it to `pkg/monitoring/metrics.go`, but not sure it is comfy here.. 
what do you say?